packages/flutter/lib/src/widgets/reorderable_list.dart

```diff
@@ -1120,13 +1117,10 @@ class _ReorderableItemState extends State<_ReorderableItem> {
       return SizedBox.fromSize(size: size);
     }
     _listState._registerItem(this);
-    return LayoutBuilder(builder: (BuildContext context, BoxConstraints constraints) {
-      _childLayoutConstraints = constraints;
-      return Transform(
-        transform: Matrix4.translationValues(offset.dx, offset.dy, 0.0),
-        child: widget.child,
-      );
-    });
+    return Transform(
+      transform: Matrix4.translationValues(offset.dx, offset.dy, 0.0),
+      child: widget.child,
+    );
```

Comment:
While we're here, maybe we could also use the specialized constructor (per the [style guide](https://github.com/flutter/flutter/blob/a0c04534f5ead3e9f0d6a861e07d8e32c7318919/docs/contributing/Style-guide-for-Flutter-repo.md#prefer-specialized-functions-methods-and-constructors)):

```dart
return Transform.translate(offset: offset, child: widget.child);
```